fs/fs_zenfs.cc

```diff
@@ -1185,7 +1186,8 @@ Status NewZenFS(FileSystem** fs, const std::string& bdevname, std::string byteda
     return Status::IOError(zbd_status.ToString());
   }
 
-  auto metrics = std::make_shared<BytedanceMetrics>(metrics_factory, bytedance_tags, logger);
+  auto metrics = std::make_shared<BDZenFSMetrics>(metrics_factory, bytedance_tags, logger);
```

Comment:
We should remove all Bytedance related code inside zenfs, so we should construct the `BDZenFSMetrics` outside (maybe in `env/env_zenfs.cc`)